tests/helpers/test_trigger.py

```diff
@@ -45,7 +37,7 @@ async def test_trigger_variables(hass: HomeAssistant) -> None:
     """Test trigger variables."""
 
 
-async def test_if_fires_on_event(hass: HomeAssistant, calls) -> None:
+async def test_if_fires_on_event(hass: HomeAssistant, service_calls) -> None:
```

Comment:
```suggestion
async def test_if_fires_on_event(
    hass: HomeAssistant,
    service_calls: list[ServiceCall]
) -> None:
```

Reply:
Good spot 👍 
I wish Github would auto-run ruff-format when applying suggestions, but I don't think that's likely to happen any time soon.